src/routes/admin.tsx

```diff
@@ -1,7 +1,55 @@
-export default function AdminPage() {
+import React, { useState } from "react";
+import { LIMIT, useUsers } from "../domain/get-users";
+
+export const AdminPage: React.FC = () => {
+  const [offset, setOffset] = useState(0);
+  const [page, setPage] = useState(1);
+
+  const { users } = useUsers(offset);
+
+  const handleNextPage = () => {
+    setPage(page + 1);
+    setOffset(offset + LIMIT);
+  };
+
+  const handlePrevPage = () => {
+    setPage(page - 1);
+    if (page != 1) {
```

Comment:
```suggestion
    if (page !== 1) {
```

É legal sempre usar a comparação estrita, pra evitar possível resultados inesperados com esses ifs.